imbaquiz.front/src/components/Answers.jsx

```diff
@@ -0,0 +1,151 @@
+// Answers.jsx
+import React, { useState, useEffect } from "react";
+import { useAuth0 } from "@auth0/auth0-react";
+import axios from "axios";
+
+const Answers = ({ questionId }) => {
+  const { getAccessTokenSilently } = useAuth0();
+  const [answers, setAnswers] = useState([]);
+  const [newAnswerText, setNewAnswerText] = useState("");
+  const [isCorrectAnswer, setIsCorrectAnswer] = useState(false);
+  const [editingAnswer, setEditingAnswer] = useState(null);
+  const [editedText, setEditedText] = useState("");
+  const [editedCorrect, setEditedCorrect] = useState(false);
+
+  useEffect(() => {
+    const fetchAnswers = async () => {
+      if (!questionId) return;
+      const token = await getAccessTokenSilently();
+      try {
+        const res = await axios.get(`https://localhost:7280/api/answers/by-question/${questionId}`, {
```

Comment:
Would be great to move the API call logic to separate files and get the API url from .env instead of hardcoding it